ansible/roles/certbot/tasks/main.yml

```diff
@@ -58,22 +66,15 @@
     --agree-tos
     --email "domreg@svsticky.nl"
     --keep-until-expiring
-    --cert-name {{ item.name }}
-    --webroot
-    --webroot-path /var/www/acme-challenges
-    --domain {{ item.name }}
-    {% if 'staging' not in group_names %}
-    {% for hostname in item.alternative_names %}
-    --domain {{ hostname }}
-    {% endfor %}
-    {% endif %}
+    --cert-name {{ item }}
+    --dns-digitalocean
+    --dns-digitalocean-credentials /etc/letsencrypt/certbot-creds.ini
+    --domain "*.{% if 'staging' in group_names %}dev.{% endif %}{{ item }}"
```

Comment:
note: to make this work, every domain should get an extra domain record (`dev.domainame` pointing to the staging server).